src/advanced/entry_points_overriding.md

```diff
@@ -1,29 +1,30 @@
 # Override entry point
 
-^Sylvia is still developing and lacks features like f.e. `sudo` support.
-If you need to use a lacking feature of `CosmWasm` or prefer to define some custom
-entry point, it is possible to use the `#[sv::override_entry_point(...)]` attribute.
+It may happen that for any reason CosmWasm will start support some new
+entry point that is not yet implemented in ^sylvia. There is a way to
+add it manually using `#[sv::override_entry_point(...)]` attribute.
+This feature can be used to override already implemented entry points
+like `execute` and `query`.
 
 ## Example
 
-To make ^sylvia generate multitest helpers with `sudo` support, you first need to define your
+To make ^sylvia generate multitest helpers with `cusom_entrypoint` support, you first need to define your
 `entry point`.
 
 ```rust,noplayground
 #[entry_point]
-pub fn sudo(deps: DepsMut, _env: Env, _msg: SudoMsg) -> StdResult<Response> {
+pub fn cusom_entrypoint(deps: DepsMut, _env: Env, _msg: SudoMsg) -> StdResult<Response> {
     CounterContract::new().counter.save(deps.storage, &3)?;
     Ok(Response::new())
 }
 ```
 
-You have to define the `SudoMsg` yourself, as it is not yet supported.
+You have to define the `CustomEntrypointMsg` yourself, as it is not yet supported.
```

Comment:
Maybe `CustomSudoMsg` instead?

Reply:
I didn't want to refer to sudo here as it is fully supported by Sylvia. It should be treated as an extra feature that helps with any new CosmWasm entrypoints that will appear in the future and won't be handled by Sylvia at that point.